results.tex

```diff
@@ -94,14 +94,14 @@ \subsection{Nuclear Material Inventory}
 \subsection{French \gls{SFR} Deployment}
 
 Reprocessing \gls{UNF} collected from all EU nations can start approximately
-180 \glspl{SFR}. Table \ref{tab:pu} lsits the isotope, mass fraction,
+180 \glspl{SFR}. Table \ref{tab:pu} lists the isotope, mass fraction,
 and quantity of plutonium that can be obtained from the 2050 \gls{UNF} inventory.
  With the \gls{SFR} breeding ratio above one, France can transition into
 a fully \gls{SFR} fleet without extra construction of \glspl{LWR}. 
 
 From Varaine et al. \cite{varaine_pre-conceptual_2012}, a French
-ASTRID-type 600\gls{MWe} \gls{SFR} consumes $1.225$ tons of
-plutonium a year, with an initial plutonium loading of $4.9$ tons. 
+ASTRID-type 600\gls{MWe} \gls{SFR} consumes $1.225$ tonnes of
```

Comment:
why tonnes? The general rule stated by most journals is that a paper can be submitted in british english or american english, but not a combination of both.

Reply:
If you want to be specific and american, you could say "metric tons"

Reply:
My understanding was that `tonne` is another metric for metric ton. But  if it is confusing, I'll switch to metric ton!